kill-stuck-pr-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+rm -f *.prop
+
+if [ "X${UPLOAD_UNIQUE_ID}" = "X" ] ; then exit 0 ; fi
+if [ "X${PULL_REQUEST}" = "X" ] ; then exit 0 ; fi
+
+REPOSITORY=$(echo ${PULL_REQUEST} | cut -d '#' -f 1)
+PR_ID=$(echo ${PULL_REQUEST} | cut -d '#' -f 2)
+
+COMMIT_ID=$(curl -L http://localhost/SDT/jenkins-artifacts/pull-request-integration/${UPLOAD_UNIQUE_ID}/prs_commits.txt | grep "^${PULL_REQUEST}=")
+if [ "X${COMMIT_ID}" = "X" ] ; then exit 0 ; fi
+
+./cms-bot/update-commit-statuses-matching.py -r ${REPOSITORY} -c ${COMMIT_ID} -p ${CONTEXT} rocm
```

Comment:
if this script runs on cmssdt node then use `${CMS_BOT_DIR}` instead of `./cms-bot/`

Reply:
@iarspider , use `${CMS_BOT_DIR}` here